lib/zendesk_apps_tools/deploy.rb

```diff
@@ -66,22 +66,22 @@ def find_app_id(product_name = 'v2') # use the v2 endpoint if no product name is
       say_status 'Update', 'app ID is missing, searching...'
       app_name = get_value_from_stdin('Enter the name of the installed app:')
 
-      response = cached_connection.get("/api/#{product_name}/apps/installations.json")
-      installations_json = json_or_die(response.body)
+      response = cached_connection.get("/api/#{product_name}/apps/owned.json")
+      owned_apps_json = json_or_die(response.body)
 
-      unless response.success? && installations_json.has_key?('installations')
+      unless response.success? && owned_apps_json.has_key?('apps')
         say_error_and_exit "Unable to retrieve installations. Please check your credentials and internet connection."
```

Comment:
Update to `Unable to retrieve apps. Please check your credentials and internet connection.`

Reply:
Good eyes.